grpc/binlog/v1alpha/binarylog.proto

```diff
@@ -31,8 +31,14 @@ message GrpcLogEntry {
     SEND_TRAILING_METADATA = 2;
     SEND_MESSAGE = 3;
     RECV_INITIAL_METADATA = 4;
+    // On server side, this event means the client has half-closed.
+    // On client side, this is means the RPC has now ended.
     RECV_TRAILING_METADATA = 5;
     RECV_MESSAGE = 6;
+    // Client side only.
+    CLIENT_SEND_CANCEL = 7;
```

Comment:
I assume that this corresponds to an HTTP/2 RSET_STREAM?

Reply:
Correct, this corresponds to the client application requesting an explicit cancellation. But because binary logging happens at a high level, I think we should treat the RST_STREAM as an implementation detail.